packages/inventory/src/components/table/InventoryTable.js

```diff
@@ -118,7 +123,7 @@ const InventoryTable = forwardRef(({
                         paginationProps={paginationProps}
                     />
                 </TableToolbar>
-            </Fragment>
+            </Fragment> : (customErrorState ? customErrorState : <ErrorState />)
```

Comment:
You can put ErrorState as the default prop for customErrorState so there is no need for conditional logic.

Also, what about callimg it just `errorState`, it’s prop, so it’s custom by default. 😄

Reply:
Nice suggestion! Updated.